web/templates/gear_menu_popover.hbs

```diff
@@ -119,6 +119,14 @@
                     </a>
                 </li>
                 {{/if}}
+                {{#if is_development_environment}}
```

Comment:
Is it easy to make this `development_environment and (is_owner or billing_administrator)`? That will make it possible to test the intended workflows.

Reply:
Pushing an extra commit for this.